kitchen-sink/react/pages/Messages.jsx

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable react/no-array-index-key */
+import { React, useState } from 'react';
+import {
+  Page,
+  Navbar,
+  NavbarBackLink,
+  Messagebar,
+  Messages,
+  Message,
+  MessagesTitle,
+  Link,
+  Icon,
+} from 'konsta/react';
+import { CameraFill, ArrowUpCircleFill } from 'framework7-icons/react';
+import { MdCameraAlt, MdSend } from 'react-icons/md';
+
+export default function MessagesPage() {
+  const isPreview = document.location.href.includes('examplePreview');
+  const [messageText, setMessageText] = useState('');
+  const [messagesData, setMessagesData] = useState([
+    {
+      type: 'sent',
+      text: 'Hi, Kate',
+    },
+    {
+      type: 'sent',
+      text: 'How are you?',
+    },
+    {
+      name: 'Kate',
+      type: 'received',
+      text: 'Hi, I am good!',
+      avatar: 'https://cdn.framework7.io/placeholder/people-100x100-9.jpg',
+    },
+    {
+      name: 'Blue Ninja',
+      type: 'received',
+      text: 'Hi there, I am also fine, thanks! And how are you?',
+      avatar: 'https://cdn.framework7.io/placeholder/people-100x100-7.jpg',
+    },
+    {
+      type: 'sent',
+      text: 'Hey, Blue Ninja! Glad to see you ;)',
+    },
+    {
+      type: 'sent',
+      text: 'How do you feel about going to the movies today?',
+    },
+    {
+      name: 'Kate',
+      type: 'received',
+      text: ' Oh, great idea!',
+      avatar: 'https://cdn.framework7.io/placeholder/people-100x100-9.jpg',
+    },
+    {
+      name: 'Kate',
+      type: 'received',
+      text: ' What cinema are we going to?',
+      avatar: 'https://cdn.framework7.io/placeholder/people-100x100-9.jpg',
+    },
+    {
+      name: 'Blue Ninja',
+      type: 'received',
+      text: 'Great. And what movie?',
+      avatar: 'https://cdn.framework7.io/placeholder/people-100x100-7.jpg',
+    },
+  ]);
+
+  const handleSendClick = () => {
+    const text = messageText.replace(/\n/g, '<br>').trim();
+    const type = 'sent';
+    const messagesToSend = [];
+    if (text.length) {
+      messagesToSend.push({
+        text,
+        type,
+      });
+
+      console.log(messagesToSend);
+    }
+    if (messagesToSend.length === 0) {
+      return;
+    }
+    setMessagesData([...messagesData, ...messagesToSend]);
+    setMessageText('');
+  };
```

Comment:
After message is sent, scroll page content to the bottom so we can see that message